types/index.d.ts

```diff
@@ -1,3 +1,4 @@
 declare var __DEV__: boolean;
 
-type MapboxValues = undefined | string | number | boolean | null | MapboxValues[];
+type MapboxValues = undefined | string | number | boolean | null | MapboxExpr;
```

Comment:
It feels weird to have two types like that, I'd rather refact the former one like that:

```
type MapboxValues = Array<undefined | string | number | boolean | null | MapboxValues>;
```

Reply:
We need both (`textColor`).